src/components/players/details/quickStats/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import Line from '../../../_shared/line';
+import Title from '../../../_shared/title';
+import Table from '../../../_shared/table';
+
+import styles from './index.module.scss';
+
+const QuickStats = ({ quickStats }) => {
+    const arr = Object.entries(quickStats);
+
+    const cell = arr.map((x, i) => {
```

Comment:
give proper names to your variables. Don't use `x` or `i`, give it a human name and for `i` call it `index`